osidb/tasks.py

```diff
@@ -0,0 +1,72 @@
+import logging
+
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django.utils import timezone
+from django_celery_results.models import TaskResult
+
+from config.celery import app
+
+logger = logging.getLogger(__name__)
```

Comment:
Since this module contains celery tasks I would use rather `celery.utils.log.get_task_logger` instead of the basic `logging.getLogger` so we can utilize the task ID and task name in the log message.